src/main/java/edu/stanford/protege/search/lucene/tab/ui/QueryResultsPanel.java

```diff
@@ -69,7 +69,7 @@ public QueryResultsPanel(OWLEditorKit k, JButton b) {
      * @param editorKit OWL Editor Kit
      */
     public QueryResultsPanel(OWLEditorKit editorKit) {
-        visibilityTimer = new Timer(200, e -> searchProgressBar.setVisible(true));
+        //visibilityTimer = new Timer(200, e -> searchProgressBar.setVisible(true));
```

Comment:
for sure, I didn't mean to push this yet,  I'm still working on it. It's this issue with the progress bar not showing during indexing. The main problem is that the various panels and views are only initialized when exposed. If the user kicks off indexing then switches to the search panel it registers it's listener too late.